spec/outputs/http_spec.rb

```diff
@@ -1 +1,232 @@
 require "logstash/devutils/rspec/spec_helper"
+require "logstash/outputs/http"
+require "thread"
+require "sinatra"
+
+PORT = rand(65535-1024) + 1025
```

Comment:
we should try to add a helper to devutils to do that, is a very common pattern in many plugins. What do you think?